php/frankenphp.Dockerfile

```diff
@@ -64,7 +65,8 @@ RUN <<EOF
         xml \
         yaml \
         zip \
-        zstd
+        zstd \
+        xsl
```

Comment:
the packages are alphabetical ordered at the moment... and x is not behind z

Reply:
Done